### DISCUSSION TITLE
Kotlin language by default uses 2 spaces indent so would it be a better idea to change to 2 spaces indent?

### DISCUSSION THREAD

**Discussion by chengenzhao** [AlmasB/FXGL#1026] [General]
Hi:

I double checked with Kotlin community and seems like they use 2 spaces indent by default
and seems like the source code of FXGL are using 4 spaces indent
so is it a good idea to change to 2 spaces indent?
The idea always prompt to apply Kotlin code style...

Thanks

**Comment by AlmasB** [comment]
Hi,

According to [Kotlin docs](https://kotlinlang.org/docs/coding-conventions.html#indentation),

> Use four spaces for indentation. Do not use tabs.

which FXGL does follow.
